ReactiveCocoa/Swift/Action.swift

```diff
@@ -44,6 +44,16 @@ public final class Action<Input, Output, Error: ErrorType> {
 	/// Whether the instantiator of this action wants it to be enabled.
 	private let userEnabled: PropertyOf<Bool>
 
+	/// Lazy creation and storage of a UI bindable CocoaAction.
+	///
+	/// The default version force casts the AnyObject? input from CocoaAction to
+	/// match `Input` but this property can be overriden for alternative behavior.
```

Comment:
Poor wording, I meant re-assigned. However, this warning is mostly moot if the `private` function works.